packages/swapper/src/swappers/thorchain/getThorTradeQuote/getTradeQuote.ts

```diff
@@ -17,63 +20,24 @@ export const getThorTradeQuote = async ({
   deps: ThorchainSwapperDeps
   input: GetTradeQuoteInput
 }): Promise<TradeQuote<ChainId>> => {
-  const { sellAsset, buyAsset, sellAmount, sellAssetAccountNumber, wallet } = input
+  const { sellAsset, buyAsset, sellAmount, sellAssetAccountNumber, wallet, chainId } = input
```

Comment:
curious looking at this implementation in regards to `sellAssetAccountNumber`, I am wondering why this is something we need to pass in from the client when it should be able to be determined by the via address/xpub using `getAccount()` in chain adapters. I feel like this would end up needing to happen in the web client in this setup when it would be nice to reduce the need for extra calls from the client if we have the necessary data to derive in lib ideally. Thoughts?

Reply:
Or the `bip44Params` even better

Reply:
for sure can probably be done that way